README.md

```diff
@@ -0,0 +1,245 @@
+| `RUN_ID`            | GitHub run ID. The ID will be a unique part of the URL address. If a new file with the same name is uploaded to the same container and with the same ID, the new file will replace the old one. We recommend using a combination of github.run_number, start time, and sha | **Yes**  | N/A     |
+
+### Expiration period environment variables
+The following variables must be passed as 
+[GitHub Actions vars context](https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values)
+or [GitHub Actions environment variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+These environment variables are responsible for the storage time of the results in the storage network in epochs 
+(in the mainnet, an epoch is approximately equal to one hour, so we can assume that values are specified in HOURS).
+
+After the period is over, the data will be deleted. They are convenient to use for log rotation or test reports.
+
+They default to 0, in which case the data will be stored until they are manually deleted.
+We recommend setting a reasonable and convenient for work expiration period, for example, a month (744 hours).
+
+For results from releases, there is no expiration date, they will be stored until they are manually deleted.
+
+| Key                            | Value                                                                                                                                                                             | Required | Default |
+|--------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `PR_EXPIRATION_PERIOD`         | Expiration period for artifacts created as a result of [opening or modifying a PR](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) | **No**   | 0       |
+| `MASTER_EXPIRATION_PERIOD`     | Expiration period for artifacts created as a result of [master/main branch modification](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push)   | **No**   | 0       |
+| `MANUAL_RUN_EXPIRATION_PERIOD` | Expiration period for artifacts created as a result of [manually run](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)         | **No**   | 0       |
+| `OTHER_EXPIRATION_PERIOD`      | Expiration period for artifacts created as a result of other events                                                                                                               | **No**   | 0       |
```

Comment:
A single expiration can still be better. Let the user calculate it in any way he wants to (not forcing this model of PR/master/whatever upon him), but have just one variable for the action.

Reply:
> A single expiration can still be better. Let the user calculate it in any way he wants to (not forcing this model of PR/master/whatever upon him), but have just one variable for the action.

Fixed